Go/Go.sublime-syntax

```diff
@@ -132,7 +132,7 @@ contexts:
     # highlighting them as compiler pragmas could be misleading. We scope them
     # as plain comments by default, and add some detailed meta scopes for
     # enterprising users wishing to add more color.
-    - match: (//)([a-z]+)(:)({{directive}})[ ]?
+    - match: (//)([a-z]+[a-z-]*)(:)({{directive}})[ ]?
```

Comment:
Are underscores supported? They are at least part of `{{ident}}`.

Reply:
There is no formal documentation for them, but there are no underscore-containing namespace cases in the wild. There are underscore-containing directives (the compiler recognises a bunch of `//go:cgo_[^ ]+`.

Given that this covers what exists, it's probably OK to leave it until golang/go#43776 is resolved.

Reply:
OK, just wanted to be sure we don't miss something.